cpp/ql/src/semmle/code/cpp/stmts/Stmt.qll

```diff
@@ -1798,7 +1798,7 @@ class FunctionTryStmt extends TryStmt {
  * }
  * ```
  */
-class CatchBlock extends Block {
+class CatchBlock extends BlockStmt {
```

Comment:
I'd like to draw attention to the fact we're not renaming related classes like `CatchBlock` here (which could become `CatchBlockStmt`).  I'm happy with this decision because I don't think `CatchBlock` is going to be confused with `BasicBlock`.